lua-linter/action.yml

```diff
@@ -0,0 +1,63 @@
+name: "Scalingo Lua linter GitHub Action"
+description: "Lint Lua codebases with lua-language-server"
+
+inputs:
+  path:
+    description: "Path passed to lua-language-server --check"
+    required: false
+    default: "."
+  base-ref:
+    description: "Deprecated: kept for backward compatibility; no effect when linting the entire repository"
+    required: false
+    default: ""
+  checklevel:
+    description: "lua-language-server check level"
+    required: false
+    default: "Hint"
+  lua-path:
+    description: "Value to export as LUA_PATH before linting"
+    required: false
+    default: ""
```

Comment:
question: I find it annoying that it's sometimes mandatory to specify `path` and `lua-path`. Isn't there a way yo make these paths defintion completely automatic?